lib/src/frontend/fixture.dart

```diff
@@ -50,6 +53,22 @@ class NgTestFixture<T> {
     activeTest = null;
   }
 
+  /// Return a page object representing [pageObjectType] from the DOM.
+  Future/*<T>*/ getPageObject/*<T>*/(Type pageObjectType) async {
```

Comment:
how about something like `findPageObject`?

`get` adds zero information to a method name - is there any more descriptive word we can use?